lib/dndtoc.sty

```diff
@@ -28,4 +28,10 @@
     {\thecontentslabel :~ } % numbered-entry-format
     {} % numberless-entry-format
     {\titlerule*{.}\contentspage} % filler-page-format
+
+  \newcommand\tocchapapp{\tocchapterabbreviationname}
+  \g@addto@macro\appendix{%
+    \addtocontents{toc}{\protect\renewcommand{\protect\@chapapp}{\appendixname}}%
```

Comment:
This line is not strictly necessary, but it allows people to use `\@chapapp` and `\chaptertitlename` in their TOC, should they ever whish to do so.

The behaviour without this line is that `\@chapapp` is always `Chapter` in that context, which tripped me up at first.